src/app/public/features/components/repository/profile/profile.component.css

```diff
@@ -0,0 +1,44 @@
+* {
+  font-weight: 800;
+  text-align: center;
+}
+
+.card {
+  margin: 0 auto;
+  padding: 10px;
+  margin-top: 15px;
+  margin-bottom: 15px;
+  border-radius: 20px;
+  background-color: black;
```

Comment:
card is too generic a name... also you should follow the BEM nomenclature. (same for button, titles...)